### PR TITLE
Cleanup 20230923

### PR DESCRIPTION
**Pull request by LucasHocker**
- Fix typo 
- Apply upstream updates
- Apply requested change from ADES-3542

### CODE REVIEW

File: src/app/components/modals/InstallOneAgentModal.tsx [hunk 1]
```diff
@@ -124,6 +123,29 @@ export const InstallOneAgentModal = ({ modalOpen, onDismiss, ips, cloudType }: I
     }
   };
 
+  const attentionLabel = (text: string) => {
```
Comment (ffriedl89) [inline src/app/components/modals/InstallOneAgentModal.tsx:126]: Separate this into a standalone component. 
```
const AttentionLabel = (props: AttenionLabelProps) => {
  const { text } = props; 
  ...
}
```

**Review by ffriedl89 — CHANGES_REQUESTED** [review]
Only minor topics, overall the changeset looks good :)

**Comment by LucasHocker** [comment]
@stefankoegl FYI this PR was pending approval for 5 months after requested changes were made.

**Review by ffriedl89 — APPROVED** [review]
The changes related to my requested changes look good to me.

**Pull request merged**